code/modules/paperwork/contract.dm

```diff
@@ -0,0 +1,265 @@
+/* For employment contracts and infernal contracts */
+
+/obj/item/weapon/paper/contract
+	throw_range = 3
+	throw_speed = 3
+	var/signed = 0
+	var/datum/mind/target
+
+/obj/item/weapon/paper/contract/proc/update_text()
+	return
+
+/obj/item/weapon/paper/contract/update_icon()
+	return
+
+
+/obj/item/weapon/paper/contract/employment
+	icon_state = "paper_words"
+
+/obj/item/weapon/paper/contract/employment/New(atom/loc, mob/living/nOwner)
+	. = ..()
+	if(!nOwner || !nOwner.mind)
+		qdel(src)
+		return -1
+	target = nOwner.mind
+	update_text(nOwner)
+
+
+/obj/item/weapon/paper/contract/employment/update_text()
+	name = "paper- [target] employment contract"
+	info = "<center>Conditions of Employment</center><BR><BR><BR><BR>This Agreement is made and entered into as of the date of last signature below, by and between [target] (hereafter referred to as SLAVE), and Nanotrasen (hereafter referred to as the omnipresent and helpful watcher of humanity).<BR>WITNESSETH:<BR>WHEREAS, SLAVE is a natural born human or humanoid, posessing skills upon which he can aid the omnipresent and helpful watcher of humanity, who seeks employment in the omnipresent and helpful watcher of humanity.<BR>WHEREAS, the omnipresent and helpful watcher of humanity agrees to sporadically provide payment to SLAVE, in exchange for permanant servitude.<BR>NOW THEREFORE in consideration of the mutual covenants herein contained, and other good and valuable consideration, the parties hereto mutually agree as follows:<BR>In exchange for paltry payments, SLAVE agrees to work for the omnipresent and helpful watcher of humanity, for the remainder of his or her current and future lives.<BR>Further, SLAVE agrees to transfer ownership of his or her soul to the loyalty department of the omnipresent and helpful watcher of humanity.<BR>Should transfership of a soul not be possible, a lien shall be placed instead.<BR>Signed,<BR><i>[target]</i>"
+
+
+/obj/item/weapon/paper/contract/employment/attack(mob/living/M, mob/living/carbon/human/user)
```

Comment:
Adding NOBLUDGEON flag to the contract and adding ..() to attack procs would be better here.